documentation/changelogs/api/2023.03.22.04.56.29.md

```diff
@@ -0,0 +1,33 @@
+# 2023.03.22.04.56.29
+
+## Improvements
+
+- Absorb `build-nginx` job into `build-images` job (#944) @dhruvkb
+- Change post status to 'publish' (#980) @dhruvkb
+- Swap from flake8 to ruff (#963) @glowatsk
+
+## Internal Improvements
+
+- Update Nuxt to v.2.16.3 (#952) @obulat
+- Configure ingestion_server as a known first party for isort (#965) @raiyaj
+- Skip build and publish job if nothing to do (#977) @dhruvkb
+- Change post status to 'publish' (#980) @dhruvkb
+- Fix needs check on api staging deployment (#975) @sarayourfriend
+- Add issues to the new Project and set Priority field value (#869) @obulat
+- Swap from flake8 to ruff (#963) @glowatsk
+
+## Bug Fixes
+
+- Switch to internal header on single results (#981) @zackkrida
+- Add GH_TOKEN to the gh steps (#984) @obulat
+- Add decoding of the strings that don't have backslashes (#979) @obulat
+- Update to new link for pipenv install instructions (#962) @kk311y
+- Treat any non 200 status as failure for thingiverse (#940) @sarayourfriend
+- Skip build and publish job if nothing to do (#977) @dhruvkb
+- Make `searchTerm` for VAudioTrack and VImageCell optional (#955) @obulat
```

Comment:
Should this be part of the API changelog? It doesn't appear to edit any API files nor does it have the API stack label 🤔

Reply:
As noted in the commit that adds these files, they're just here for testing: https://github.com/WordPress/openverse/pull/987/commits/1a757d19bee7b4f57068a48e105132affe81355f

I copied the file from the existing draft release, it has nothing to do with the release-drafter workflow configurations introduced in this PR.